core/components/atoms/switch/switch.js

```diff
@@ -113,14 +124,17 @@ Switch.propTypes = {
   /** Labels to show, import for accessibility */
   accessibleLabels: PropTypes.arrayOf(PropTypes.string),
   /** Locked switch */
-  readOnly: PropTypes.bool
+  readOnly: PropTypes.bool,
+  /** Label on left side */
+  labelLeft: PropTypes.bool
```

Comment:
Can we change the `labelLeft` to `labelDirection: ['left', 'right']` with default as 'right'?

Reply:
Of course! I will try to do that now